packages/typespec-ts/test/integration/generated/routes/src/index.d.ts

```diff
@@ -363,12 +390,18 @@ export declare interface QueryParametersQueryContinuationExplodeArray204Response
 
 export declare type QueryParametersQueryContinuationExplodeArrayParameters = QueryParametersQueryContinuationExplodeArrayQueryParam & RequestParameters;
 
+export declare interface QueryParametersQueryContinuationExplodeArrayParamQueryParam {
+    value: string[];
+    explode: true;
+    style: "form";
+}
+
 export declare interface QueryParametersQueryContinuationExplodeArrayQueryParam {
     queryParameters: QueryParametersQueryContinuationExplodeArrayQueryParamProperties;
 }
 
 export declare interface QueryParametersQueryContinuationExplodeArrayQueryParamProperties {
-    param: string[];
+    param: QueryParametersQueryContinuationExplodeArrayParamQueryParam;
```

Comment:
This looks like a breaking change, but it isn't, right? We didn't support this scenario until now?